tools/validators/instance_validator/validate/subscriber.py

```diff
@@ -19,7 +19,7 @@
 from concurrent import futures
 import json
 
-from google.auth import jwt
+import google.auth
```

Comment:
can you please keep the previous form? our internal linter will not accept the new suggestion